docs/README.md

```diff
@@ -11,26 +11,25 @@
 - [Dumping](#dumping)
 
 ## Core concepts
-**clin** sends http requests to Nakadi to create or update resources. The source
-of data for this is yaml manifest file which corresponds to a single Nakadi
+**clin** sends HTTP requests to Nakadi to create or update resources. The source
+of data for this is YAML manifest file which corresponds to a single Nakadi
 resource.
 
 These manifests can be applied:
 - directly one by one, see
-  [Applying single manifest](#applying-single-manifest).
+  [Applying a single manifest](#applying-single-manifest).
 - in batch by using a "clin file" to roll out whole infrastructure with a single
   command, see [Batch processing](#batch-processing).
 
 `{{TEMPLATE_VARIABLES}}` and `@@@./includes.yaml` , optionally enforced by
-[yaml anchors](https://www.google.com/search?q=yaml+anchors) enable expressive
+[YAML anchors](https://www.google.com/search?q=yaml+anchors) enable expressive
 and DRY definition of the Nakadi infrastructure.
 
 For more details see [Manifests format](#manifests-format).
 
 ### Dry-run by default
-Each command that can possibly change Nakadi resources in NOT performing any
-changes by default. To apply actual changes, additional key `-X` or `--execute`
-should be specified:
+Commands do NOT apply any changes by default. To apply changes, 
+additional key `-X` or `--execute` should be specified:
```

Comment:
`the` additional key, probably also `the` changes. Though maybe consider rephrasing to something like this:

To apply the changes, pass the `-X` or `--execute` flag to the command: